### PR TITLE
Add token filters

### PR DESCRIPTION
**Pull request by bonustrack**
~Develop branch is behind, we should merge this PR first: https://github.com/balancer-labs/pool-management/pull/199~

### CODE REVIEW

**Comment by Destiner** [comment]
Overall looks solid. Few comments:

* It doesn't list DAI and USDC. This is likely because token picker uses `CreatePoolForm` store. I think we can add similar state vars to `Pool` store.
* It should reset input on modal open (or close), similar to how it works in pool creation (but again, it should use separate state).
* When you navigate (e.g. open a pool) while filtering applied, and go back to the pool list, filtering is still applied, but UI doesn't show it. We should either reset filtering or update UI (whatever is the easiest to implement).

**Comment by bonustrack** [comment]
@Destiner Thanks, fixed the issues also added autofocus on search input and loading status.

File: src/components/Filters/index.tsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
+import React, { useState } from 'react';
+import { RouteComponentProps } from 'react-router';
```
Comment (Destiner) [inline src/components/Filters/index.tsx:2]: I believe this line is not used which makes linting fail.
```suggestion
```

**Comment by Destiner** [comment]
Great work @bonustrack! I think we're ready to merge.

**Pull request merged**